Sources/Auth/Apple/AppleAuthenticator.swift

```diff
@@ -11,20 +11,15 @@ import Foundation
 import PovioKitAuthCore
 import PovioKitPromise
 
-public protocol AppleAuthProvidable: AuthProvidable {
-  func signIn(from presentingViewController: UIViewController,
-              with nonce: AppleAuthenticator.Nonce) -> Promise<Response>
-  func checkAuthState() -> Promise<AuthProvidable.Authenticated>
-}
-
 public final class AppleAuthenticator: NSObject {
-  private let userIdStorageKey = "povioKit.appleSocialProvider.signIn.userId"
-  private let storage: UserDefaults = .standard
+  private let userIdStorageKey = "signIn.userId"
+  private let storage: UserDefaults
   private let provider: ASAuthorizationAppleIDProvider
   private var processingPromise: Promise<Response>?
   
   public override init() {
     self.provider = .init()
+    self.storage = .init(suiteName: "povioKit.appleAuthenticator") ?? .standard
```

Comment:
Should we allow the user to initialise the `storage`?

Reply:
I think we should yes.